.github/CODEOWNERS

```diff
@@ -9,7 +9,15 @@
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
 *  @calebmshafer
-/packages/imodel-select  @wilmaier
+
+/packages/imodel-select  @wilmaier @calebmshafer
+/common/changes/@bentley/imodel-select-react  @wilmaier @calebmshafer
+
 /packages/markup-frontstage  @JoeZman @archana-maharjan
-/packages/tree-widget  @JoeZman @diegopinate @aruniverse @JValiunas @michaalx
+/common/changes/@bentley/markup-frontstage-react @JoeZman @archana-maharjan
+
 /packages/property-grid  @JoeZman @aruniverse
+/common/changes/@bentley//property-grid  @JoeZman @aruniverse
```

Comment:
should be: `/common/changes/@bentley/property-grid`

Reply:
you have an extra `/`

Reply:
also, can we add diego as a code owner for this one?

Reply:
I had fixed that but forgot to save the file before I committed :)  Pushing now.

Reply:
Add @diegopinate and fixes the path issue.